src/Data/ProgrammesDb/Entity/Ancestry.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity\EntityInterfaces\MemberOfGroupInterface;
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity\EntityInterfaces\PromotableInterface;
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity\EntityInterfaces\RelatedLinkContextInterface;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+
+/**
+ * @ORM\Entity(repositoryClass="BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ImageRepository")
+ */
+class Ancestry implements MemberOfGroupInterface, RelatedLinkContextInterface, PromotableInterface
+{
+    use TimestampableEntity;
+    use Traits\IsEmbargoedTrait;
+    use Traits\PartnerPidTrait;
+    use Traits\SynopsesTrait;
```

Comment:
I think that we should be able to remove all of these traits as they apply to specific columns being available for the table.